### PR TITLE
[DS-1310] Add and enable the CKEditor 5 Paste Filter module

### PR DESCRIPTION
**Pull request by aleevas**
Original Issue, this PR is going to fix: [DS-1310](https://yusa.atlassian.net/browse/DS-1310)

Make sure these boxes are checked before asking for review of your pull request - thank you!

If there is a new feature or this is a bug fix - use `main` branch. We'll tag for release if the bug is critical ASAP or tag for release next bug fix release until the critical issue arrives.

## Steps for review

- [ ] log as admin
- [ ] go to /admin/config/content/formats/manage/full_html
- [ ] verify that **Paste filter** is enabled
image

- [ ] go to /node/add/article_lb
- [ ] paste formatted text (heading, font color, color background, font size, font family etc) from MS Word or LibreOffice Write to body field
- [ ] verify that some styles are cleared
image
- [ ] save the node
- [ ] verify that the text has cleared formats (some)
- [ ] verify that others nodes and blocks in LB

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer. See [Code of Conduct](https://github.com/ymcatwincities/openy/wiki/Open-Y-Code-of-Conduct-and-Best-Practices).
- [ ] Try to use Conventional commit messages accordingly to the standard https://www.conventionalcommits.org/en/v1.0.0/#specification
- [ ] [Documentation](https://github.com/ycloudyusa/yusaopeny/tree/main/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ycloudyusa/yusaopeny/blob/main/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with an account on drupal.org, otherwise, you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ycloudyusa/yusaopeny/blob/main/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -369,9 +370,6 @@
                 "Issue #3215817: Presentational attributes such as 'border', 'align', or 'bgcolor' are used. CSS should be used for styling instead.": "https://git.drupalcode.org/project/video_embed_field/-/merge_requests/3.diff",
                 "Add support for Ckeditor 5, see https://www.drupal.org/project/video_embed_field/issues/3311063#comment-15227318": "https://www.drupal.org/files/issues/2023-09-12/3311063-35.patch"
             },
```
Comment (aleevas) [inline composer.json:372]: we have duplication of this patch https://github.com/YCloudYUSA/yusaopeny/blob/a63d1c181ce52689225221e0e78cbb5960d283ae/composer.json#L372

**Comment by rollki** [comment]
test this please

**Comment by rollki** [comment]
test this please

**Comment by rollki** [comment]
@aleevas 
Paste filter is not enabled by default, pls check it
Screenshot 2024-06-07 at 10 11 02
http://profile.openy.cibox.tools/build4831/admin/config/content/formats/manage/full_html

**Review by rollki — APPROVED** [review]
Changes LGTM!

**Comment by podarok** [comment]
retest this please

**Pull request merged**